cookbooks/hll_zeppelin.md

```diff
@@ -43,7 +43,7 @@ spark.udf.register("hll_cardinality", HllCardinality)
 
 ## Example Usage
 
-This is a short example which can also be used to verify expected behavior.
+This is a short example which can also be used to verify expected behaviour.
```

Comment:
Let's not favo[u]r the British spelling.

Reply:
You mean Canadian? :)

`markdown-spellcheck` comes with a US dictionary as well, but it seems to be missing a lot more sort-of-jargon words so I went with the default (which was, surprisingly, *not* the US version).

I will change it in favor the US dictionary.

Reply:
Does the US dictionary allow both spellings or just the US spelling? I'd suggest allowing both.

Reply:
The US dictionary is just as picky about the superfl[u]o[u]s u's.

The spelling library's docs say:
> We default to British English but will change in the next major to American.

So I think we're better off and more future-proof by using the US dictionary.  I've updated the spellings accordingly.